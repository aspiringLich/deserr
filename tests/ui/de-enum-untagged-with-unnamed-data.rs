use deserr::DeserializeFromValue;

#[derive(DeserializeFromValue)]
#[deserr(error = deserr::Error)]
enum Enum {
    EmptyVariant,
    VariantWithSomething(u16),
}

fn main() {}
