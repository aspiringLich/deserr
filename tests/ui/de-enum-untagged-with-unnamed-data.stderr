error: Enum variants with unnamed associated data aren't supported by the Deserr derive macro.
 --> tests/ui/de-enum-untagged-with-unnamed-data.rs:7:25
  |
7 |     VariantWithSomething(u16),
  |                         ^^^^^
